packages/services/f-statistics/src/index.js

```diff
@@ -0,0 +1,23 @@
+/* eslint-disable camelcase */
+import defaultOptions from './defaultOptions';
```

Comment:
the export name is `defaultConfig`

Reply:
Ill pick this one up on the next bump - won't cause an impact; but could be renamed to follow convention